doc/installation/system/inifile.rst

```diff
@@ -112,6 +112,16 @@ If ``PI_HSM_PADDING`` is not set, the default internal padding is used::
 .. warning:: The two paddings are not compatible. Data encrypted with one padding can not decrypted with the other padding. 
    So if you change the padding later during the cause of operation, it will not be possible to decrypt old data!
 
+``PREFERRED_LANGUAGE`` is a list in which the preferred languages can be defined. The browser's language settings are compared to this
+list and the "best match" wins. If none of the languages set in the browser match, the first language in the list will be used as the output language::
+
+    PREFERRED_LANGUAGE = ["en", "de", "es", "fr"]
+
+.. note:: If ``PREFERRED_LANGUAGE`` is not defined, a ``DEFAULT_LANGUAGE_LIST`` is used.
+   DEFAULT_LANGUAGE_LIST = ["en", "de", "de_DE", "nl", "fr", "cs", "es", "tr", "zh_Hant"]
+
+   If none of these languages are defined in the browser, the first language in the list will also be used here.
```

Comment:
I think this should be clear from the text above.

Reply:
fixed in 09d11230e87bdf58510ca5d85aed0f8271538b71.